src/core/read_halos-velociraptor.c

```diff
@@ -105,28 +105,6 @@ void read_trees__velociraptor(int snapshot,
   // TODO: For the moment, I'll forgo chunking the read.  This will need to
   // be implemented in future though, as we ramp up the size of the trees
```

Comment:
We can remove this comment at this point

Reply:
done